style/index.css

```diff
@@ -152,8 +165,9 @@ div.CodeMirror-linebackground.jp-InputArea-editor-dependencyline {
 }
 
 .jp-OutputArea-highlighted.jp-OutputArea-selected .jp-OutputArea-gatherbutton {
-  box-shadow: inset 0 0 10px var(--brand-color0);
   background: var(--brand-color3);
+  background-blend-mode: multiply;
```

Comment:
Just curious, what is `background-blend-mode` set for?

Reply:
Ah, that was for making the button a little more visible if there was text behind it, or if a line behind the button was highlighted in purple. Now that opacity is set to 1, this can be removed! Will change in the next commit.

Reply:
Cool! Can you add that commit into this PR?